now-mainnet.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 2,
+  "public": true,
+  "scope": "aragon",
+  "alias": "network.aragon.org",
+  "build": {
+    "env": {
+      "ARAGON_ETH_NETWORK_TYPE": "main",
+      "ARAGON_FORTMATIC_API_KEY": "@network-dashboard-fortmatic-api-key",
+      "ARAGON_PORTIS_DAPP_ID": "@network-dashboard-portis-dapp-id",
+      "ARAGON_SENTRY_DSN": "@network-dashboard-sentry-dsn"
```

Comment:
These don't exist yet, do we want to be using new ones?

Reply:
I think the names are fine, but we will have to create them on Vercel.